cli/node/service.py

```diff
@@ -52,3 +52,17 @@ def get_peers(self):
         return {
             'peers': peers,
         }, None
+
+    def get_info(self):
+        """
+        Get node information.
+        """
+        try:
+            information = loop.run_until_complete(self.service.node_management.get_node_info())
```

Comment:
Change `information` to `node_information` or `nodes_information`, because now it not is clear.

Reply:
@Alladin9393, done.